### PR TITLE
Nz/2676 508 forms

### LINKED ISSUE #2676: [Dev] [508] Verify and update forms to render value to screen reader
As noted in the 508 compliance review:
> Form value (whatever the user is typing) should also be rendered by screen reader. The input text box should have role=value.

More investigation should be done with this issue before moving forward with a solution.

Additional notes:
* We are using the design.cms.gov Text field - https://design.cms.gov/components/text-field/
* Using Safari + Voice Over, the form field values are being rendered by the screen reader

### This task is done when…
This could include:
 - [ ] A clear understanding of the issue is discovered
 - [ ] A new ticket is created with a clear solution to be implemented

### PR DESCRIPTION
**Pull request by Sun-Mountain**
Resolves #2676 

### Description
Confirming that screen readers pick up the input fields' text when selected.

I added the `role` attribute when missing.

### Significant changes or possible side effects
I have noted known bugs when using Mac's Voiceover in Chrome in the [eAPD Wiki](https://github.com/CMSgov/eAPD/wiki/Accessibility-in-the-Quality-Process#known-bugs).

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| User | using a screen reader | the existing text in an input will be read outloud | accessibility |

### Steps to manually verify this change

1. Go thru the page with a screen reader and confirm that all inputs gives the value of the input. (If using Chrome on a Mac, there may be issues with [known bugs](https://github.com/CMSgov/eAPD/wiki/Accessibility-in-the-Quality-Process#known-bugs)

### This pull request is ready to code review when

- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [x] Associated OpenAPI documentation has been updated
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/static/schemas/plannedActivityShedule.js [hunk 1]
```diff
@@ -1,16 +1,12 @@
 const Joi = require('joi').extend(require('@joi/date'));
 
 const scheduleSchema = Joi.object({
```
Comment (thetif) [inline web/src/static/schemas/plannedActivityShedule.js:3]: this needs to be moved to the common package.

**Review by thetif — APPROVED** [review]
Why are we getting so many prettier issues again? I was hoping I had fixed it.

Just need to move the schema to the common package.

File: common/schemas/costAllocation.js [hunk 2]
```diff
@@ -8,4 +8,4 @@ const costAllocateSchema = Joi.object({
   })
 });
 
-export default costAllocateSchema;
```
Comment (tbolt) [inline common/schemas/costAllocation.js:11]: shouldn't the prettier pre-commit hook catch these?

**Review by tbolt — APPROVED** [review]

**Review by mirano-darren — APPROVED** [review]
Using NVDA all known issues documented does not happen. Thinking this could be a bug with Chrome + VoiceOver (like shouty box). This looks good to me!

**Review by jeromeleecms — APPROVED** [review]
👍

File: common/schemas/costAllocation.js (hunk 2 above)
Comment (tbolt) [reply common/schemas/costAllocation.js:11]: @Sun-Mountain following up on this. I mis-understood the diff. I originally thought the end-of-file newline was being removed but it was being added (which is correct and prettier would take care of that)

**Pull request merged**